scrapy/utils/reqser.py

```diff
@@ -47,7 +50,11 @@ def request_from_dict(d, spider=None):
     eb = d['errback']
     if eb and spider:
         eb = _get_method(spider, eb)
-    return Request(
+    try:
```

Comment:
What has been the thought process here to use a try except block? Why not:
```python
 request_cls = load_object(d['_class']) if '_class' in d else Request
```
This makes more sense to me since `d['_class']` is only set when `type(request) is not Request` and since `d['_class']` stores import path of request, `load_object (d['_class'])` should not error. Or is there a case when it will throw an error which I'm unaware of?